src/helpers/hendleErorrs.ts

```diff
@@ -0,0 +1,11 @@
+import { Errors } from '../enums/Errors';
+
+export const handleErrors = (
```

Comment:
Naming convention can be different from project to project but here are some basic rules:

`handle` prefix - used when handling DOM events like `onChange`, onBlur`

`on` prefix - used for component props like `onTodoAdd`, `onTodoDelete`, `onError`

Helpers function should start with some verbs like `get`, `format` and something like this